### ISSUE TITLE
Expected top-1 test accuracy

### ISSUE THREAD

**Issue by xidulu** [abarcel/haikumodels#1]
Hi

This is a fantastic project! The released checkpoints are super helpful!

I am wondering what's the top-1 test accuracy that one should get using the released ResNet-50 checkpoints. I am able to reach 0.749 using the my own ImageNet dataloader implemented via Tensorflow Datasets. Is the number close to your results?

BTW, it would also be very helpful if you could release your training and dataloading code for these models!

Thanks,

**Comment by abarcel** [comment]
Hi and thanks;

I didn't write these models with the purpose of using on ImageNet, but it was actually a part of different project.

Expected accuracies should be really close to be what to be expected from Keras Applications, so you can directly take their available model accuracy list as a reference. I would recommend using loading your data via TensorFlow Datasets to keep it simple, this is also that I'm doing.

**Comment by xidulu** [comment]
Thanks!
